server/db/index.js

```diff
@@ -5,6 +5,15 @@ const Conversation = require('./models/conversation');
 const Activity =  require('./models/activity');
 
 //TODO: associations
+Reply.belongsTo(Conversation);
+Conversation.hasMany(Reply);
+
+User.hasMany(Reply);
+
+Activity.belongsTo(User);
+User.hasMany(Activity);
+
+Activity.belongsTo(Reply);
```

Comment:
Could activity belong to a conversation as well assuming it's the beginning of a thread?

Reply:
I updated this -- activity has a third type "post"
If you think about the views where this will be listed out, such as on a user's profile page or a report, I think it will be easier to just get all the activities and then filter by type of post/reply/like